sql-plugin/src/main/spark320/scala/org/apache/spark/sql/rapids/shims/RapidsErrorUtils.scala

```diff
@@ -87,4 +88,8 @@ object RapidsErrorUtils extends RapidsQueryErrorUtils
   def tableIdentifierExistsError(tableIdentifier: TableIdentifier): Throwable = {
     QueryCompilationErrors.tableIdentifierExistsError(tableIdentifier)
   }
+
+  def invalidInputSyntaxForBooleanError(s: UTF8String): RuntimeException = {
```

Comment:
how about for shims of 32x ?

Reply:
I tested locally on 321, 330 and 341 and the tests pass. PTAL